### ISSUE TITLE
Crash when hiding "stickers" and "call buttons" UI elements together

### ISSUE THREAD

**Issue by b02860de585071a2** [rhunk/SnapEnhance#492]
### Description

Hiding both the "stickers" and "call buttons" UI elements causes Snapchat to crash when attempting to open a conversation. 

Crash seems to be a NPE. This only started with the December A14 patch, as far as I can tell.

---

### System info:

- Pixel 8 Pro (husky)
- Android 14 (UQ1A.231205.015)
- Snapchat versions 12.63.0.55 and 12.64.0.42 tested
- SnapEnhance version 1.2.5 (arm64-v8a variant from GitHub releases)
- LSPosed version 1.9.2

Snapchat logcat attached, LSPosed logs are available if needed. 

[2023_12_12-09-01-34_450.log.zip](https://github.com/rhunk/SnapEnhance/files/13650999/2023_12_12-09-01-34_450.log.zip)

### Reproduction steps

1. Go to `Hide UI Elements`
2. Select `Stickers` and `Call Buttons`
3. Attempt to open chats
4. See error.

### Logs

```markdown
AndroidRuntime: Shutting down VM


1702396919.473 10597 25560 25560 I LSPosed-Bridge: Crash unexpectedly: java.lang.NullPointerException: 

Attempt to invoke virtual method 'void android.view.View.setOnClickListener(android.view.View$OnClickListener)' on a null object reference
```

### Snapchat Version

12.63.0.55, 12.64.0.42

### Agreement

- [X] I am using the latest stable SnapEnhance version.
- [X] This is not a bug regarding Snapchat+.
- [X] I have provided a detailed description of the issue.
- [X] I have attached a log if deemed neccessary.
- [X] This issue is not a duplicate.

**Comment by b02860de585071a2** [comment]
Well, it turns out this may be fixed(?) in the 2.0.0 debug release. 

I'm going to assume it has something to do with "call" buttons being split into chat/profile buttons?

Leaving this open until I have confirmation, just in case.

**Comment by rhunk** [comment]
Yes it has been fixed in 2.0.0
